node-fs-writefile/cp.js

```diff
@@ -0,0 +1,14 @@
+import fs from 'node:fs/promises';
+
+const [, , f1, f2] = process.argv;
+
+async function copyToFile() {
+  const f1Text = await fs.readFile(f1, 'utf8');
+  await fs.writeFile(f2, f1Text, { flag: 'a+' });
```

Comment:
When copying a file you usually wouldn't append it, but would replace. The default flag value ('w') works great for this.